packages/itinerary-body/src/defaults/route-description-footer.tsx

```diff
@@ -1,24 +1,38 @@
 import React, { ReactElement } from "react";
-
+import { differenceInMinutes } from "date-fns";
+import { Leg } from "@opentripplanner/types";
 import * as S from "../styled";
-import * as mock from "../__mocks__/route-description-footer/route-description-footer-props.json";
 import { RouteDescriptionFooterProps } from "../types";
 
 export default function DefaultRouteDescriptionFooter({
-  leg,
-  arrivalText = mock.default.arrivalText,
-  arrivalTripId = leg.tripId,
-  navigateToArrivalVehicle = () => null
+  navigateToArrivalVehicle = () => null,
+  waitMinutes
 }: RouteDescriptionFooterProps): ReactElement {
-  const { tripId } = leg;
-  return arrivalText && tripId === arrivalTripId ? (
+  return (
     <S.ArrivalTimeContainer
       onClick={() => {
         navigateToArrivalVehicle();
       }}
     >
-      <S.TrackerIcon />
-      {arrivalText}
+      Arrives in {waitMinutes} minutes
     </S.ArrivalTimeContainer>
-  ) : null;
+  );
+}
+
+export function makeRouteDescriptionFooter(): (props: {
+  leg: Leg;
+}) => JSX.Element {
+  const NewComponent = ({ leg }: { leg: Leg }) => {
+    const toTime = leg.to.arrival;
+    const fromTime = leg.from.arrival;
+    const waitMinutes = differenceInMinutes(
+      new Date(toTime),
+      new Date(fromTime)
+    );
```

Comment:
This is the travel time between the leg's from and to locations, no the wait time. Is this for illustrative purposes, or did I miss a detail?

Reply:
Yes, I just wanted it to be illustrative.